examples/typescript-reactflux/.tslintrc

```diff
@@ -0,0 +1,53 @@
+{
+  "rules": {
+    "class-name": true,
+    "curly": true,
+    "eofline": true,
+    "forin": true,
+    "indent": [true, 4],
+    "label-position": true,
+    "label-undefined": true,
+    "max-line-length": [true, 140],
+    "no-arg": true,
+    "no-bitwise": false,
+    "no-console": [true,
+        "debug",
+        "info",
+        "time",
+        "timeEnd",
+        "trace"
+    ],
+    "no-construct": true,
+    "no-debugger": true,
+    "no-duplicate-key": true,
+    "no-duplicate-variable": true,
+    "no-empty": false,
+    "no-eval": true,
+    "no-string-literal": true,
+    "no-switch-case-fall-through": true,
+    "no-trailing-comma": true,
+    "no-trailing-whitespace": false,
+    "no-unused-expression": false,
+    "no-unused-variable": true,
+    "no-unreachable": true,
+    "no-use-before-declare": true,
+    "one-line": [false,
+        "check-open-brace",
+        "check-catch",
+        "check-else",
+        "check-whitespace"
+    ],
+    "quotemark": [false, "double"],
+    "radix": true,
+    "semicolon": true,
+    "triple-equals": [false, "allow-null-check"],
+    "variable-name": false,
+    "whitespace": [false,
+        "check-branch",
+        "check-decl",
+        "check-operator",
+        "check-separator",
+        "check-type"
+    ]
+  }
+}
```

Comment:
I would prefer not to have a typescript linting setup in this PR. We will have one at one point, but that's a separated topic as it envolves more then one app.